docs/main.js

```diff
@@ -1 +1,22 @@
-console.log("hI");
\ No newline at end of file
+document.querySelector('form').addEventListener('submit', function(event) {
+    const name = document.getElementById('name').value.trim();
+    const email = document.getElementById('email').value.trim();
+    const message = document.getElementById('message').value.trim();
+
+    if (!name || !email || !message) {
+        event.preventDefault();
+        alert('Please fill out all the fields!');
+        if (!name) document.getElementById('name').classList.add('error');
```

Comment:
I wanted to see a bit more alteration of the HTML content directly, but this works.